### DISCUSSION TITLE
GPX track doesn't appear

### DISCUSSION THREAD

**Discussion by LaurentL06** [bozdoz/wp-plugin-leaflet-map#201] [Q&A]
Hello,
I have recently been using the Leaflet Map extension in WordPress. In a code module, the map appears correctly. But when I want to add one or more GPX tracks to this map, this or these tracks do not appear on the map. I converted the GPS file to GEOJSON, I have the same malfunction on the map: no trace appears [!]
I don't understand. My code is :
`[leaflet-map lat=45.752008274262785 lng=2.9676224598940024 zoom=10 zoomcontrol !detect-retina]
[leaflet-gpx src=https://www.alpes-sud-photos.com/dev/wp-content/uploads/2023/01/GR4_ETAPE_24.gpx color=red]`
And in Leaflet Map parameters module, I have https://unpkg.com/@mapbox/togeojson@0.16.0/togeojson.js adress for JavaScript converter for KML/GPX.

Do you have a solution. Thanks for your help.

**Comment by bozdoz** [comment]
Don't have a solution, sorry
